src/rules/valid-title.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  createRule,
+  getNodeName,
+  getNodeTitle,
+  isDescribe,
+  isTestCase,
+} from './utils';
+
+function trimFXprefix(word: string) {
+  return ['f', 'x'].includes(word.charAt(0)) ? word.substr(1) : word;
+}
+
+export default createRule({
+  name: __filename,
+  meta: {
+    docs: {
+      category: 'Best Practices',
+      description: 'Enforce valid titles',
+      recommended: false,
+    },
+    messages: {
+      duplicatePrefix: 'should not have duplicate prefix',
+      accidentalSpace: 'should not have space in the beginning',
+    },
+    type: 'suggestion',
+    schema: [],
+  },
+  defaultOptions: [],
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (!isDescribe(node) && !isTestCase(node)) return;
+
+        const title = getNodeTitle(node);
+        if (!title) return;
+
+        if (title.charAt(0) === ' ') {
```

Comment:
You could use `trimStart` here to detect any & all spaces - I can't imagine any title that'd you want to start w/ multiple spaces, a la how you'd not want one space.